maps.css

```diff
@@ -0,0 +1,17 @@
+/* 
```

Comment:
![image](https://github.com/apu52/Travel_Website/assets/114172928/d5159f20-6b7c-42fe-8350-9d2101a982b5)

I can see there is a new view button for the google maps but whenever I am clicking it , it simply redirects me to a blank page .....

Reply:
@apu52  yes because we need google map api .and it will charge 2 rupees but if it exceeds a limit of 200 dollars it would be counted .
So you please generate api key google cloud platform from your end .

Reply:
ok then Can you please guide me in the API matter